tests/many_to_many/tests.py

```diff
@@ -28,6 +30,12 @@ def setUp(self):
         self.a4 = Article.objects.create(headline='Oxygen-free diet works wonders')
         self.a4.publications.add(self.p2)
 
+        self.t1 = Tag.objects.create(name='web')
```

Comment:
It's a bit odd that you made changes in the setUp method but didn't actually use that data in any tests. Is it automatically tested somehow?